File: rb/utility/make_context.rb
# EnergyChartsApi2 SDK utility: make_context
require_relative '../core/context'
module EnergyChartsApi2Utilities
  MakeContext = ->(ctxmap, basectx) {
    EnergyChartsApi2Context.new(ctxmap, basectx)
  }
end

File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// EnergyChartsApi2 SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class EnergyChartsApi2MakeContext
{
    public static function call(array $ctxmap, ?EnergyChartsApi2Context $basectx): EnergyChartsApi2Context
    {
        return new EnergyChartsApi2Context($ctxmap, $basectx);
    }
}
